.github/workflows/pr-labeler.yaml

```diff
@@ -1,7 +1,6 @@
 name: PR Labeler
 
 on: 
-  pull_request:
```

Comment:
@markostreich @devtobi 

Das führt dazu, dass es für interne PRs nicht mehr geht😐

Reply:
@darenegade @devtobi 
[Hier](https://github.com/it-at-m/digiwf-core/pull/1533) ging das. Oder was übersehe ich?

Reply:
https://github.com/it-at-m/digiwf-core/pull/1516

Ich hatte es hier auch schon als ersten Commit getestet und es ging erst nicht, daher hatte ich `pull_request` in diesem PR wieder reingenommen. Aber keine Ahnung warum es nun doch so geht🤷‍♂️